core/model/src/main/java/org/eclipse/rdf4j/model/datatypes/XMLDatatypeUtil.java

```diff
@@ -676,6 +681,15 @@ public static boolean isValidAnyURI(String value) {
 		}
 	}
 
+	public static boolean isValidLanguage(String value) {
```

Comment:
I am bit off today, the main part of this function is already in core/model in Literals, but currently it throws an exception rather than returning false:

https://github.com/eclipse/rdf4j/blob/master/core/model/src/main/java/org/eclipse/rdf4j/model/util/Literals.java#L587

Good location for this function could be beside that in Literals